website/frontend/templates/docs/scripting.html

```diff
@@ -89,20 +144,31 @@ <h3 id="left_text_num">$left(text,num)</h3>
           </li>
         </ul>
 
+        <h3 id="right_text_num">$right(text,num)</h3>
+
+        <ul>
+          <li>Returns the last
```

Comment:
Nit pick away - I have no problems with fixing stuff that wasn't included in my changes - after al if I had spotted it during the edit I would have changed it anyway.